src/lib/chawathe98_matcher.rs

```diff
@@ -1450,18 +1421,16 @@ impl<T: Clone + Debug + Eq + 'static> MatchingTreesScriptor<T> for Config2 {
                     break;
                 }
             }
-            if !vector_input.is_empty(){
+            if !vector_input.is_empty() {
                 end_copy_tree.insert(vector_input[vector_input.len() - 1]);
             }
         }
         // Add Nodes Copy Tree
         for nodes in end_copy_tree {
             if store.check_subtree(nodes.0, nodes.1) {
-                /*
-                - If insertion in sub-tree from tree 2
-                - If deletion in sub-tree from tree 1
-                    -> Then it means the glue operations has occurred.
-                */
+                //                - If insertion in sub-tree from tree 2
```

Comment:
Is there a reason for the huge amounts of space after `//`?

Reply:
Fixed. I think it was because I did it automatically, instead of manually.